logstash-core/lib/logstash/plugins/mixin.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+
+module LogStash
+  module Plugins
+    module Mixin
+
+      def annotate(name="")
+        if !name.empty?
+          super("filter", name)
```

Comment:
Why does this only work to set "filter" type plugins? I assume that this mixin is only called directly like filters?